packages/components/src/FilterBar/_docs/FilterBar.mdx

```diff
@@ -341,3 +341,45 @@ and <a href="https://www.npmjs.com/package/query-string">query-string</a>
 packages.
 
 <NoClipCanvas of={FilterBarStories.ExternalEventValuesUpdate} />
+
+### Open a filter through selecting a value from another filter
+
+There may be a situation where you want to open a filter based on selecting a value from another filter.
+
+To do so, call the `openFilter` function from `useFilterBarContext` in your event handler
+(you will need to create a custom component wrapping the Filter to access the context).
+
+You may also want to combine the usage with an `isUsableWhen`, as a selected value increases the width of
+the Filter button (therefore pushing the others), potentially misaligning the popover of the one opened if it has relocated.
```

Comment:
Hard to understand the connection between these two statements, how does `isUsableWhen` affect the popover alignment?